userguide/tasks.rst

```diff
@@ -1757,8 +1757,8 @@ shows the configuration options for Cloud Syncs.
    |                     |                     |                                                                                                         |
    +---------------------+---------------------+---------------------------------------------------------------------------------------------------------+
    | Credential          | drop-down menu      | Select the cloud storage provider credentials from the list of available :ref:`Cloud Credentials`.      |
-   |                     |                     | The UI automatically tests the credential and displays an error if a connection cannot be made.         |
-   |                     |                     | :guilabel:`Save` is disabled until a valid Credential is entered.                                       |
+   |                     |                     | The credential is tested and an error is displayed if a connection cannot be made. :guilabel:`Save` is  |
+   |                     |                     | disabled until a valid Credential is entered.                                                           |
```

Comment:
The last "Credential" should be lower case.

Reply:
done